src/refiner/development.sig

```diff
@@ -27,15 +27,15 @@ sig
     type theorem
     type operator_decl
 
-    datatype t =
+    datatype 'a t =
```

Comment:
Can we call this `'w` or `'u` or something? (Just so I don't forget what it is for)